src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <title>Leader Board</title>
+</head>
+<body>
+    <nav class="navbar navbar-expand-md navbar-light bg-light mt-5">
+        <h4 class="text-dark header">LEADERBOARD EX</h4>
+    </nav>
+
+    <div class="card container-sm p-5 appWrapper">
+    <div class="d-flex appContainer">
+        <div class="dynamicSec d-col">
+            <div class="d-flex"> <h3 id="heading">Recent Scores </h3>  <button id="refresh" class="btn btn-outline-dark">Refresh</button>  </div>
+            <div id="dynamicDisplay">
+
+            </div>
+        </div>
+        <div class="additionForm">
+            <form class="d-col">
+                <label for="name">Full Name</label>
+                <input type="text" name="name" id="name">
+                <label for="score">Your Score</label>
+                <input type="number" name="score" id="score">
+                <input type="button" value="Submit" class="btn btn-outline-dark mt-2">
+                </form>
+        </div>
+    </div>
```

Comment:
- [ ] It is requested in the project requirement that your plain HTML markup should be styled with minimum styling to look like the provided framework (see image). Therefore as suggested in the previous review, kindly make sure your design looks like the provided wireframe to follow the project requirement.
![Screenshot from 2023-08-02 09-04-13](https://github.com/paulshaun-kdtk/leaderboard/assets/91142432/a5f3b661-f0c4-4e8c-9b7b-a59a79c82365)